docs/source/conf.py

```diff
@@ -216,6 +217,10 @@
     "og_image": "https://voxel51.com/wp-content/uploads/2024/03/3.24_webpages_Home_AV.png",
 }
 
+# -- Options for pushfeedback extension ---------------------------------------
+pushfeedback_project = "1nx7ekqhts"
```

Comment:
I assume this is public information that anybody could get by looking at the webpage?

Just want to make sure we haven't included a secret here, and I'm not familiar enough with pushfeedback to make that decision on my own...